signac/common/config.py

```diff
@@ -55,12 +58,22 @@ def search_tree(root=None):
             root = up
 
 
-def search_standard_dirs():
+def _search_standard_dirs():
```

Comment:
Do we need to deprecate `search_tree` and the other permissions-related functions in this file? Can’t recall if we’ve also removed those in 2.0 or if they should be removed.

Reply:
Those were already changed in next, but whoever made those changes originally didn't bother to deprecate and I wasn't sure whether to go ahead and add those deprecations here. I can do that now if you'd like.

Reply:
I don't think deprecations are necessary -- let's just remove them.